client/src/pages/search/index.js

```diff
@@ -20,40 +24,91 @@ export default function Search() {
+      video_id: "iOZVbILaIZc",
+    },
+    {
+      id: 116,
+      start_time: 1034.1079999999995,
+      text: "AUDIENCE: [INAUDIBLE] JOHN GUTTAG: Exactly.You store the answer and then look it up when you need it. Because we know that we can look things up very quickly. Dictionary, despite what Eric said in his lecture, almost all the time works in constant time if you make it big enough, and it usually is in Python. We'll see later in the term how to do that trick. ",
+      video_id: "uK5yvoXnkSk",
+    },
+    {
+      id: 1409,
+      start_time: 1383.9119999999994,
+      text: "This is, having written a book recently, the editor in charge of books for Prentice Hall.And I can assure you that data processing is a fad that won't last out the year.\" MIT had that attitude for a while. For about 35 years, computer science was in a building off campus, because they weren't sure we were here to stay. Maybe that's not why, but that's why I interpret it. Ken Olsen, an MIT graduate-- I should say, a course 6 graduate-- was the founder and president and chair of Digital Equipment Corporation, which in 1977 was the second largest computer manufacturer in the world based in Maynard, Massachusetts. ",
+      video_id: "iOZVbILaIZc",
+    },
+    {
+      id: 1182,
+      start_time: 340.58400000000006,
+      text: "Now if we look at this, and we use K nearest neighbors, those are the three nearest to the first numeral, and they are all fours.And so we vote and we decide it's a zero. Is it infallible? No. But it's typically much more reliable than just nearest neighbors, hence used much more often. And that was our problem, by the way, with the alligator. ",
+      video_id: "eg8DJYwdMyg",
+    },
+  ];
   return (
     <>
       <Header />
       <Head>
         <title>Video Search</title>
       </Head>
+      
       <div className="bg-zinc-900 text-zinc-200 h-screen py-32">
-        <div className="flex flex-col gap-10 items-center p-6">
+        <div className="flex flex-col gap-10 items-center p-6 ">
           <SearchBar />
 
-          <div className="flex flex-col items-center w-full">
-            {isLoading ? (
-              <Spinner />
-            ) : (
-              data.map((ele) => (
-                <div key={ele.id}>
-                  <VideoElement
-                    video_id={ele.video_id}
-                    start={ele.start_time}
-                    text={ele.text}
-                  />
-                </div>
-              ))
-            )}
+          <div className="px-24 flex flex-wrap items-center w-full">
```

Comment:
we need only one video per column and in to be exactly in the middle of the page.